libparsec/crates/client/tests/unit/certificates_ops/poll.rs

```diff
@@ -0,0 +1,220 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use libparsec_client_connection::{
+    test_register_low_level_send_hook, test_register_send_hook, HeaderMap, ResponseMock, StatusCode,
+};
+use libparsec_protocol::authenticated_cmds;
+use libparsec_tests_fixtures::prelude::*;
+use libparsec_types::prelude::*;
+
+use crate::certificates_ops::PollServerError;
+
+use super::utils::certificates_ops_factory;
+
+#[parsec_test(testbed = "minimal", with_server)]
+async fn empty(env: &TestbedEnv) {
+    let alice = env.local_device("alice@dev1");
+
+    let ops = certificates_ops_factory(env, &alice).await;
+
+    test_register_send_hook(
```

Comment:
If you register a request hook, then you don't need the `with_server` given you won't talk to the server ! ^^

Reply:
Done